jitsi/020-deployment.yaml

```diff
@@ -0,0 +1,153 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: jitsi
+  namespace: jitsi
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    k8s-app: jitsi
+  name: jitsi
+  namespace: jitsi
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      k8s-app: jitsi
+  template:
+    metadata:
+      labels:
+        k8s-app: jitsi
+    spec:
+      containers:
```

Comment:
Each of these can be split into their own pod. You can reference pods in the same namespace for networking purposes by using the name of the pod (separate namespaces can use `pod_name.namespace_name`).

Reply:
Don't forget to make the Services point to the correct Pod when they are renamed.